documentation/package-table.md

```diff
@@ -1,40 +1,42 @@
+| **Linux-musl-x64** | [![][linux-musl-x64-badge-main]][linux-musl-x64-version-main]<br>[tar.gz][linux-musl-x64-targz-main] - [Checksum][linux-musl-x64-targz-checksum-main] | [![][linux-musl-x64-badge-9.0.1XX-preview6]][linux-musl-x64-version-9.0.1XX-preview6]<br>[tar.gz][linux-musl-x64-targz-9.0.1XX-preview6] - [Checksum][linux-musl-x64-targz-checksum-9.0.1XX-preview6] | [![][linux-musl-x64-badge-8.0.4XX]][linux-musl-x64-version-8.0.4XX]<br>[tar.gz][linux-musl-x64-targz-8.0.4XX] - [Checksum][linux-musl-x64-targz-checksum-8.0.4XX] |
+| **Linux-musl-arm** | [![][linux-musl-arm-badge-main]][linux-musl-arm-version-main]<br>[tar.gz][linux-musl-arm-targz-main] - [Checksum][linux-musl-arm-targz-checksum-main] | [![][linux-musl-arm-badge-9.0.1XX-preview6]][linux-musl-arm-version-9.0.1XX-preview6]<br>[tar.gz][linux-musl-arm-targz-9.0.1XX-preview6] - [Checksum][linux-musl-arm-targz-checksum-9.0.1XX-preview6] | [![][linux-musl-arm-badge-8.0.4XX]][linux-musl-arm-version-8.0.4XX]<br>[tar.gz][linux-musl-arm-targz-8.0.4XX] - [Checksum][linux-musl-arm-targz-checksum-8.0.4XX] |
+| **Linux-musl-arm64** | [![][linux-musl-arm64-badge-main]][linux-musl-arm64-version-main]<br>[tar.gz][linux-musl-arm64-targz-main] - [Checksum][linux-musl-arm64-targz-checksum-main] | [![][linux-musl-arm64-badge-9.0.1XX-preview6]][linux-musl-arm64-version-9.0.1XX-preview6]<br>[tar.gz][linux-musl-arm64-targz-9.0.1XX-preview6] - [Checksum][linux-musl-arm64-targz-checksum-9.0.1XX-preview6] | [![][linux-musl-arm64-badge-8.0.4XX]][linux-musl-arm64-version-8.0.4XX]<br>[tar.gz][linux-musl-arm64-targz-8.0.4XX] - [Checksum][linux-musl-arm64-targz-checksum-8.0.4XX] |
+| **RHEL 6** | **N/A** | **N/A** | **N/A** |
 
 Reference notes:
 > **1**: Our Debian packages are put together slightly differently than the other OS specific installers. Instead of combining everything, we have separate component packages that depend on each other. If you're installing the SDK from the .deb file (via dpkg or similar), then you'll need to install the corresponding dependencies first:
 > * [Host, Host FX Resolver, and Shared Framework](https://github.com/dotnet/runtime/blob/main/docs/project/dogfooding.md#nightly-builds-table)
 > * [ASP.NET Core Shared Framework](https://github.com/aspnet/AspNetCore/blob/main/docs/DailyBuilds.md)
 
+.NET Core SDK 2.x downloads can be found at [.NET Core SDK 2.x Installers and Binaries](Downloads2.x.md) but they are [out of support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).
+
```

Comment:
I noticed that in PR #41396, we removed some part of the code. If we still need this part of the code, then we might also need to add the [2.x installer table](https://github.com/dotnet/sdk/pull/41396/commits/8913e1f274d552ded4cff131d02533be74b6ea64).